test/spec/modules/smaatoBidAdapter_spec.js

```diff
@@ -161,17 +161,20 @@ describe('smaatoBidAdapterTest', () => {
             id: 'bidId',
             banner: BANNER_OPENRTB_IMP,
             tagid: 'adspaceId',
-            bidfloor: 0
           }
         ]);
       });
 
       it('sends bidfloor when configured', () => {
         const singleBannerBidRequestWithFloor = Object.assign({}, singleBannerBidRequest);
-        singleBannerBidRequestWithFloor.getFloor = function() {
-          return {
-            currency: 'USD',
-            floor: 0.123
+        singleBannerBidRequestWithFloor.getFloor = function(arg) {
+          if (arg.currency === 'USD' &&
```

Comment:
well, I hope that having `if` statement in tests is okay practice in js XD

Reply:
we will see :zap: